debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorTask.java

```diff
@@ -59,7 +60,10 @@ public String version() {
     @Override
     public ChangeEventSourceCoordinator start(Configuration config) {
         final Clock clock = Clock.system();
-        final MySqlConnectorConfig connectorConfig = new MySqlConnectorConfig(config);
+        final MySqlConnectorConfig connectorConfig = new MySqlConnectorConfig(
+                config.edit()
+                        .with(AbstractDatabaseHistory.INTERNAL_PREFER_DDL, true)
```

Comment:
This means we can't disable that feature for the MySQL connector, right? Wouldn't it be better to change the Field in the MySQLConnectorConfig to a default value of true. That way it's possible to deactivate it when necessary?

Reply:
We definitely don't want to deactivate. MySQL connector never worked in the past with logical schema so this is the backward compatible behaviour.